docs/source/features.rst

```diff
@@ -3,142 +3,152 @@
 
 .. image:: _static/screenshots/share.png
 
-After you add files, you'll see some settings. Make sure you choose the setting you're interested in first before you start sharing.
+After you add files, you'll see some settings. Make sure you choose the setting you're interested in before you start sharing.
 
 .. image:: _static/screenshots/share-files.png
 
-By default, as soon as someone finishes downloading your files, OnionShare will automatically stop the server, completely removing the website from the internet. If you want to allow multiple people to download these files, uncheck the "Stop sharing after files have been sent (uncheck to allow downloading individual files)" box.
+As soon as someone finishes downloading your files, OnionShare will automatically stop the server, removing the website from the Internet.
+To allow multiple people to download them, uncheck the "Stop sharing after files have been sent (uncheck to allow downloading individual files)" box.
 
-Also, if you uncheck this box, people will be able to download individual files that you share rather than a single compressed version of all of the files.
+Also, if you uncheck this box, people will be able to download the individual files you share rather than a single compressed version of all the files.
 
-When you're ready to share, click the "Start sharing" button. You can always click "Stop sharing", or quit OnionShare, to immediate take the website down. You can also click the up arrow icon in the top-right corner to show the history and progress of people downloading files from you.
+When you're ready to share, click the "Start sharing" button. You can always click "Stop sharing", or quit OnionShare, immediately taking the website down. You can also click the "↑" icon in the top-right corner to show the history and progress of people downloading files from you.
 
 .. image:: _static/screenshots/share-sharing.png
 
-Now that you have a website that's sharing files, copy the address and send it to the person you want to receive the files. If the files need to stay secure, you should use an encrypted messaging app.
+Now that you have a OnionShare, copy the address and send it to the person you want to receive the files. If the files need to stay secure, or the person is otherwise exposed to danger, use an encrypted messaging app.
 
-That person then must load the address in Tor Browser. After logging in with the random password that's included in the web address, they will be able to download the files directly from your computer by clicking the "Download Files" link in the corner.
+That person then must load the address in Tor Browser. After logging in with the random password included in the web address, the files can be downloaded directly from your computer by clicking the "Download Files" link in the corner.
```

Comment:
"Download Files" is not in the app strings…

Reply:
This string will always be in English. Only the desktop app of OnionShare is localized into different languages. The websites that people load in Tor Browser are always all in English, in order to prevent leaking the locale of the sender, in case they're trying to remain anonymous.